src/actions/queries.js

```diff
@@ -133,6 +138,16 @@ export const updateEventById = (id, event) =>
 export const getAttendanceForEvent = (eventId) =>
   axios.post("/api/attendance/statistics", { eventId });
 
+export const getSimpleAttendanceForEvent = (eventId) =>
+  axios.post("/api/attendance/simplestatistics", { eventId });
+
+//export const getEventStatistics = (startDate, endDate) =>
+export const getEventStatistics = () =>
```

Comment:
Why does this not take a start and end date anymore? On the mongo side, you should be able to easily filter the attendance objects by checkIn with something like: `{checkedIm:{$gte:startDate,$lt:endDate}`